hello.js.org

raulmatei opened the pull request:
We are a small JavaScript community called HelloJS. We'll have our first conf in June and we're preparing our site content and the entire event details these days. So, having this opportunity to use hello.js.org domain is great.

Thanks.

raulmatei committed:
Added hello.js.org

indus commented:
You have to delete the ```gh-pages``` branch (or at least the CNAME file) and trigger GitHub to rebuild the page. It is a user page and the additional CNAME interfers the forwarding

raulmatei commented:
Hi,

I've just updated the repository. Removed the CNAME file and added a landing page. We're still waiting for the final design to create a more detailed page.

Thanks.

indus commented:
But now you have no CNAME at all?!? You need one in the ```master``` branch with ```hello.js.org``` in it. I´m not in a hurry but I like to get pull-requests closed

raulmatei commented:
Sorry, my bad :). Added back the CNAME. 
As far as I can see it works now.

Many thanks.

indus commented:
Welcome [hello.js.org](http://hello.js.org). Thanks for the backlink. Nice page.
